### ISSUE TITLE
Сохраннение пользовательских regexp

### ISSUE THREAD

**Issue by zhenyatnk** [pekabon/LogGrokCore#6]
Функционал по сохранению регекспы пользователя.

Пользоватьель вбивает regexp и может сохранить его задав имя.
Пользователь может выбрать регекс из списка сохранненных и загрузить его в поле regexp

**Comment by pekabon** [comment]
История уже есть в виде примитивного intellisence, можно доделать интерфейс как на гитхабе или в браузере с выпадающим дропдауном, "звездой" в поле ввода и тд

![image](https://user-images.githubusercontent.com/4972778/146838091-f10c10e6-498e-4aa7-a458-f6631c4844e7.png)

**Comment by zhenyatnk** [comment]
да, что то подобное, главное сохранять и имя регекспа (алиас), чтобы потом можно было выбрать по имени алиаса регексп
